bin/spring

```diff
@@ -3,15 +3,18 @@
 # This file loads spring without using Bundler, in order to be fast.
 # It gets overwritten when you run the `spring binstub` command.
 
+# Run rake dev:spring to toggle spring
+ENV["DISABLE_SPRING"] = "1" if File.exists?(File.join(__dir__, "..", "tmp", "spring-off.txt"))
+
```

Comment:
Wouldn't setting the DISABLE_SPRING ENV var be enough to disable it? 🤔

Reply:
But I want to be able to enter consoles without having to set than environmental variable, and run migrations, etc.